src/libs/actions/Task.ts

```diff
@@ -15,23 +15,41 @@ import * as UserUtils from '@libs/UserUtils';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
-import {isNotEmptyObject} from '@src/types/utils/EmptyObject';
+import type * as OnyxTypes from '@src/types/onyx';
+import type {Icon} from '@src/types/onyx/OnyxCommon';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
 import * as Report from './Report';
 
-let currentUserEmail;
-let currentUserAccountID;
+type OptimisticReport = Pick<OnyxTypes.Report, 'reportName' | 'managerID' | 'participantAccountIDs' | 'notificationPreference' | 'pendingFields' | 'visibleChatMemberAccountIDs'>;
```

Comment:
Why did we define this here? Also is this generic for any type of a report or is this specifically Optimistic Task Report?